addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -288,3 +290,10 @@ def _track_subtype(self, init_values):
             return self.env.ref('stock_picking_batch.mt_batch_state')
         return super()._track_subtype(init_values)
 
+    def apply_cluster_picking(self, picking_ids, result_package_id):
```

Comment:
I think it could make sense to move this method into the `stock.picking` model,  so you can avoid to pass picking_ids as a parameter and work directly on the pickings to cluster.

Reply:
Done